kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/FormServiceBase.java

```diff
@@ -139,9 +137,12 @@ public String getFormDisplayProcess(String containerId, String processId, String
         throw new IllegalStateException("No form for process with id " + processDesc.getName() + " found");
     }
 
-    public String getFormDisplayTask(String containerId, long taskId, String lang, boolean filterContent, String formType) {
-        containerId = registry.getContainerId(containerId, new ByTaskIdContainerLocator(taskId));
```

Comment:
Is there a specific reason to delete this line? Is the functionality not needed any more?

Reply:
@pefernan I meant this line:
`containerId = registry.getContainerId(containerId, new ByTaskIdContainerLocator(taskId));`
It makes sure that correct container id is retrieved in case user uses container alias when calling this method.

Reply:
@sutaakar got it! added it again, I was missing that part.